### PR TITLE
Fixing session id to be something per user as oppose to global hard-c…

### PR DESCRIPTION
**Pull request by donnadionne**
…oded.

### CODE REVIEW

File: cpp/client.cc [hunk 1]
```diff
@@ -56,18 +56,25 @@ class WalletClient {
     ClientContext context;
     context.set_wait_for_ready(true);
     if (user == "Alice") {
-      context.AddMetadata("authorization", "2bd806c9");
+      std::string token = "2bd806c9";
+      context.AddMetadata("authorization", token);
       context.AddMetadata("membership", "premium");
+      if (affinity) {
+        // use something unique per user as session id.
+        context.AddMetadata("session_id", token);
```
Comment (dapengzhang0) [inline cpp/client.cc:64]: Are we supposed to use the same value for session_id and authorization token?

Comment (donnadionne) [reply cpp/client.cc:64]: I just picked token as it is already per user; but we can invent something else per user (so we are not confusing people in this example).  Maybe that's better?

Comment (dapengzhang0) [reply cpp/client.cc:64]: Yeah, I think that would be better. cc @menghanl who just added go support.

Comment (menghanl) [reply cpp/client.cc:64]: Yeah, let's use a different value.

Comment (donnadionne) [reply cpp/client.cc]: ok sounds good!  Fixed!  PTAL thank you!

**Review by dapengzhang0 — APPROVED** [review]
LGTM

**Review by menghanl — APPROVED** [review]

**Pull request merged**